src/domain/ecoverse/ecoverse.service.ts

```diff
@@ -407,17 +407,9 @@ export class EcoverseService {
   async removeUser(userID: number): Promise<boolean> {
     const user = await this.userService.getUserByID(userID);
     if (!user) throw new Error(`Could not locate specified user: ${userID}`);
-
-    const groups = await this.getGroups();
-    for (let i = 0; i < groups.length; i++) {
-      const group = groups[i];
-      //await this.userGroupService.removeUserFromGroup(user, group);
-    }
-
-    // And finally remove the user
+    
     await this.userService.removeUser(user);
-
-    return true;
+    return await this.accountService.removeUserAccount(user.accountUpn);
```

Comment:
should there not be a check re whether authentication is enabled?

Reply:
Yes, fixed in 0bacba83ec3a0e28251a363c9067951f2361c2b0